pkg/applicationserver/io/packages/alcsync/v1/package.go

```diff
@@ -86,22 +101,35 @@ func (a *alcsyncpkg) HandleUp(
 			continue
 		}
 		if err != nil {
-			logger.WithError(err).Debug("Failed to execute command")
+			logger.WithError(err).WithField("command_id", cmd.Code()).Error("Failed to execute command")
```

Comment:
I don't think this should be an `Error`, for the same reason - it's not a server error.